web/jobs/jobs.go

```diff
@@ -620,16 +621,27 @@ func patchJob(c echo.Context) error {
 	if _, err := jsonapi.Bind(c.Request().Body, &req); err != nil {
 		return wrapJobsError(err)
 	}
+
+	log := inst.Logger().
+		WithField("job_id", j.ID()).
+		WithField("worker_id", "client").
+		WithNamespace("jobs")
+	msg := &exec.KonnectorMessage{}
+	if err := j.Message.Unmarshal(&msg); err == nil {
+		log = log.
+			WithField("slug", msg.Konnector).
```

Comment:
Current konnector log also contain konnector version in `version` field.
but maybe it's harder to get it for client-side konnectors.